addons/post_processing/node/post_process.gd

```diff
@@ -171,14 +171,6 @@ func _process(delta):
 	if not configuration:
 		return
 	if Engine.is_editor_hint():
```

Comment:
Shouldn't Line `173` be deleted? There is a parse error for line `174` because it is not indented after the `if Engine.is_editor_hint()`. If line `173` is kept, isn't it the same as
```gdscript
	if Engine.is_editor_hint() and configuration.reload and (dynamically_update or Engine.is_editor_hint()):
```
and will never call `update_shaders()` outside the editor?

Reply:
propably

Reply:
You're right, @calvinchd , that slipped in.